pkg/resources/external_function_acceptance_test.go

```diff
@@ -2,16 +2,14 @@ package resources_test
 
 import (
 	"fmt"
-	"testing"
-
 	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
-
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider/resources"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/hashicorp/terraform-plugin-testing/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/plancheck"
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+	"testing"
```

Comment:
this should be raised as not gofumpted (this import should be with other internal imports)

Reply:
I'm pretty sure I ran `pre-push` 🤦